te_markdown/terminology.md

```diff
@@ -8,7 +8,33 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 ## Overview
 
-<!--- TODO: insert (annotated?) overview image and briefly describe the main areas of the user interface -->
+![Overview](/images/te.overview.svg?sanitize=true){:class="img-responsive"}
+
+### The different areas of the Test-Editor-Web
+In the screenshot above you can see the different areas of the Test-Editor-Web which are divided
+through the different yellow areas. The description of the specific area can be found under the number of this area underneath.   
+
+#### 1. The Test-Navigator area
```

Comment:
General remarks (applies to all of these sub-sections):
1. Consider either removing "area", or, if you think it is important in this context, capitalize it (title case).
    * I think its not needed, except for "Editor Area", because just "Editor" may be to unspecific in that case. I would argue that our name for that particular region is "Editor Area", just as "Test Navigator" is – its just its name in that case.
2. I would also consider removing the initial "The" in all cases.
3. You wouldn't normally hyphenate compound words in english, so its just "Test Navigator", without the hyphen.